pipelines/Jenkinsfile

```diff
@@ -0,0 +1,248 @@
+              githubUser=${GITHUB_USER}
+              githubToken=${GITHUB_TOKEN}
+              allow.root=true
+              org.gradle.vfs.watch=false
+              docker_com.etendoerp.docker_db=true
+              org.gradle.jvmargs=-Dfile.encoding=UTF-8" > gradle.properties
+              """
+
+              echo "---------------- Add dependencies ----------------"
+
+              def buildGradleContent = """
+                dependencies {
+                    implementation(\"com.etendoerp.platform:etendo-core:latest.release\")
+                }
+              """
+
+              sh """
+              cd etendo_base
+              echo '${buildGradleContent}' >> build.gradle
+              cat build.gradle
+              """
+              
+              echo "---------------- Build & Compile ----------------"
+              sh """
+              cd etendo_base
+              mkdir modules
+              cd modules
+              git clone https://github.com/etendosoftware/com.etendoerp.docker.git
+              cd com.etendoerp.docker
+              git checkout ${GIT_COMMIT}
+              cd ../..
+              ./gradlew prepareConfig
+              ./gradlew setup
+              ./gradlew resources.up -Pcontext.name=${CONTEXT_NAME}
+              """
+
+              echo "---------------- Verifying Docker Containers ----------------"
+              def dockerPsOutput = sh(script: 'docker ps', returnStdout: true)
+              if (!dockerPsOutput.contains("etendo-db-1")) {
+                error("The etendo-db-1 container is not running")
+              }
+
+              echo "---------------- Docker Containers are running ----------------"
+              
+              sh """
+              cd etendo_base
+              ./gradlew resources.stop -Pcontext.name=${CONTEXT_NAME}
+              """
+              
+              echo "---------------- Cleaning Docker Containers ----------------"
+              sh """
+              cd etendo_base
+              ./gradlew resources.down -Pcontext.name=${CONTEXT_NAME}
+              """
+              
+              dockerPsOutput = sh(script: 'docker ps', returnStdout: true)
+              if (dockerPsOutput.contains("etendo-db-1")) {
+                error("The etendo-db-1 container still running")
+              }
+
+              currentBuild.result = SUCCESS
+            } catch (Exception e) {
+              echo "---------------- Build environment Failed ----------------"
+              echo "Exception occurred: " + e.toString()
+              currentBuild.result = FAILED
+              error('Build Failed')
+            }
+          }
+        }
+      }
+    }
+  }
+  
+  post {
+    success {
+      container('compiler') {
+        script {
+          sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker success "Successful Bundles Test" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

Comment:
@RomanMagnoli No action is required for ending in 'success'.

Reply:
By starting as 'Pendig':
`sh './utils/repo-update-scripts/build-update-github.sh ${MODULE_PACKAGE} ${COMMIT_INPROGRESS_STATUS} "Starting build" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'`
After finishing the execution successfully, is it no needed to change it to Successful??

Reply:
It is true, we should remove the commit update instruction for post 'fixed' and keep it for 'success'.